### PR TITLE
Address Issue #5129: Document how to rename (move) project directory 

### PR DESCRIPTION
**Pull request by langaa-egeskov**
### The issue:  Document how to rename or move a project directory
I was helped by the answer in issue #5123 and the further guidance in issue #5129 

### The fix: Added to existing documentation in 2 places
- Added documentation in installation.md under 'Virtualenv' mapping caveat
- Added documentation to the virtualenv.md

This is my first pull request so feedback very welcome.

### The checklist

* [X ] Associated issue
* [ X] A news fragment in the `news/` directory to describe this fix with the extension `.bugfix.rst`, `.feature.rst`, `.behavior.rst`, `.doc.rst`. `.vendor.rst`. or `.trivial.rst` (this will appear in the release changelog). Use semantic line breaks and name the file after the issue number or the PR #.

### CODE REVIEW

File: news/5129.doc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+- Added documentation on how to move or rename a project directory
```
Comment (oz123) [inline news/5129.doc.rst:1]: Please remove leading `-`.

**Review by oz123 — CHANGES_REQUESTED** [review]
Thanks for your contribution. Only a minor request for a change.

**Comment by langaa-egeskov** [comment]
Sorry, I don't mean to leave this hanging.  I'm working on resolving why the code linting and ruff tests are failing and I haven't figured it out yet.

**Comment by matteius** [comment]
@IamAdiSri the ruff tests are a new issue because it keeps introducing new rules or inspecting new files -- it also happened on my draft-no-reqslib PR ... don't worry about it, just fix the main lint of the CI job and you'll be good.

**Comment by matteius** [comment]
You can try running: `pre-commit run --all-files --verbose --show-diff-on-failure --fix` for the CI issue preventing your build.

**Comment by langaa-egeskov** [comment]
thanks for the help! on my machine tests ran and  seems sucessful
image

**Comment by oz123** [comment]
All the failures that ruff reports have nothing to do with the PR.

**Review by oz123 — COMMENTED** [review]
Thank you for your contribution.

**Pull request merged**